.github/workflows/typescript_test.yml

```diff
@@ -53,40 +73,42 @@ jobs:
 
       - name: Set Matrix
         id: set-matrix
-      run: |
-        SUITES="[]"
-
-        # Check components changes (excluding starter projects)
-        if [[ "${{ steps.filter.outputs.components }}" == "true" ]]; then
-          SUITES=$(echo $SUITES | jq '. += ["components"]')
-        fi
-
-        # Check starter-projects changes (including components in starter projects)
-        if [[ "${{ steps.filter.outputs.starter-projects }}" == "true" ]]; then
-          SUITES=$(echo $SUITES | jq '. += ["starter-projects"]')
-        fi
-
-        # Check workspace changes
-        if [[ "${{ steps.filter.outputs.workspace }}" == "true" ]]; then
-          SUITES=$(echo $SUITES | jq '. += ["workspace"]')
-        fi
-
-        # Check API changes
-        if [[ "${{ steps.filter.outputs.api }}" == "true" ]]; then
-          SUITES=$(echo $SUITES | jq '. += ["api"]')
-        fi
-
-        # Check database changes
-        if [[ "${{ steps.filter.outputs.database }}" == "true" ]]; then
-          SUITES=$(echo $SUITES | jq '. += ["database"]')
-        fi
-
-        # If release changes or no specific changes detected, run all suites
-        if [[ "${{ steps.filter.outputs.release }}" == "true" || "$SUITES" == "[]" ]]; then
-          SUITES='["starter-projects", "components", "workspace", "api", "database"]'
-        fi
-
-        echo "suites=$SUITES" >> $GITHUB_OUTPUT
+        run: |
+          # Start with input suites if provided, otherwise empty array
+          SUITES="${{ inputs.suites || '[]' }}"
+
+          RELEASE="${{ inputs.release || 'false' }}"
+
+          if [[ "$RELEASE" == "true" ]]; then
+            SUITES='["release"]'
+          else
+            # If input suites were not provided, determine based on changes
+            if [[ "$SUITES" == "[]" ]]; then
+              # Add suites based on changed files
+              if [[ "${{ steps.filter.outputs.components }}" == "true" ]]; then
+                SUITES=$(echo $SUITES | jq '. += ["components"]')
+              fi
+              if [[ "${{ steps.filter.outputs.starter-projects }}" == "true" ]]; then
+                SUITES=$(echo $SUITES | jq '. += ["starter-projects"]')
+              fi
+              if [[ "${{ steps.filter.outputs.workspace }}" == "true" ]]; then
+                SUITES=$(echo $SUITES | jq '. += ["workspace"]')
+              fi
+              if [[ "${{ steps.filter.outputs.api }}" == "true" ]]; then
+                SUITES=$(echo $SUITES | jq '. += ["api"]')
+              fi
+              if [[ "${{ steps.filter.outputs.database }}" == "true" ]]; then
+                SUITES=$(echo $SUITES | jq '. += ["database"]')
+              fi
+
+              # If release changes or no specific changes detected, use all suites
+              if [[ "${{ steps.filter.outputs.release }}" == "true" || "$SUITES" == "[]" ]]; then
```

Comment:
you probably forgot to put the inputs release check here

Reply:
actually I dont have that release change anymore, this if should be removed